package/wpe/wpeframework/wpeframework.mk

```diff
@@ -183,14 +183,6 @@ define WPEFRAMEWORK_POST_TARGET_REMOVE_HEADERS
 	rm -rf $(TARGET_DIR)/usr/include/WPEFramework
 endef
 
-define WPEFRAMEWORK_POST_STAGING_CDM_HEADER
-	ln -sfn $(STAGING_DIR)/usr/include/WPEFramework/interfaces/IDRM.h $(STAGING_DIR)/usr/include/cdmi.h
```

Comment:
This will break the backward compatibility. There still might be a code that does not include IDRM.h and use cdmi.h instead. IMHO this should be moved to wpeframework-interfaces

Reply:
https://github.com/rdkcentral/ThunderInterfaces/pull/64/files